src/App/AboutPage.xaml

```diff
@@ -6,7 +6,7 @@
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     mc:Ignorable="d"
-    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
```

Comment:
Any specific reason to remove this ?

Reply:
Due to the way SystemResources color components in WinUI 3, adding a background brush as so would create an off-color rectangle in the middle of the frame. It looks very out of place.